lib/shopify-cli/shopifolk_check.rb

```diff
@@ -0,0 +1,45 @@
+require 'fileutils'
+
+module ShopifyCli
+  class ShopifolkCheck
+    GCLOUD_CONFIG_PATH = '~/.config/gcloud/configurations/config_default'
+    def self.assume_shopifolk?
+      ShopifyCli::ShopifolkCheck.new.assume_shopifolk?
+    end
+
+    def assume_shopifolk?
+      set_shopifolk_feature_by_gcloud_config
+      shopifolk_by_dev? && shopifolk_by_feature?
+    end
+
+    def shopifolk_by_dev?
+      File.exist?('/opt/dev/bin/dev') && File.exist?('/opt/dev/.shopify-build')
+    end
+
+    def shopifolk_by_feature?
+      is_shopifolk = ShopifyCli::Feature.enabled?('shopifolk')
```

Comment:
`'shopifolk'` should probably be a Constant at the top of this class instead of a string